setup.py

```diff
@@ -231,10 +235,10 @@ def run(self):
 forbid_publish()
 
 install_requires = []
-if pre_python26:
+if pre_PY26:
     install_requires.extend(['ssl', 'simplejson'])
 
-if sys.version_info < (3, 2):
+if pre_PY279 or pre_PY32:
```

Comment:
I was confused for a second since I thought `pre_PY32` means any version before 3.2 so this check wouldn't work, but it really means any Python 3 version which is smaller than 3.2.

Should probably rename the variables to make this more obvious.